src/app.ts

```diff
@@ -98,8 +99,9 @@ function withSpidAuth(
     next: express.NextFunction
   ) => {
     passport.authenticate("spid", async (err, user) => {
+      const issuer = getSamlIssuer(req.body);
```

Comment:
you should handle the `undefined` value, either with a default value (e.g. `UNKNOWN`) or by not logging the issuer

Reply:
fixed in d098c6c